packages/system-data/src/fields/settings.yaml

```diff
@@ -353,6 +353,134 @@ fields:
       - cast-json
     width: full
 
+  - field: reporting_divider
+    interface: presentation-divider
+    options:
+      icon: link
```

Comment:
Can we find a more suitable icon?
I'm thinking, for example, of
- https://fonts.google.com/icons?icon.set=Material+Icons&icon.query=feedback&selected=Material+Icons+Outlined%3Areviews%3A (more neutral, therefore I think also suitable for "feature")
- https://fonts.google.com/icons?icon.set=Material+Icons&icon.query=commun&selected=Material+Icons+Outlined%3Afeedback%3A

Reply:
Picked `feedback` icon, I like it.